source/core/parameters.cc

```diff
@@ -826,6 +826,10 @@ namespace Parameters
                         "1",
                         Patterns::Integer(),
                         "interpolation order tracer");
+      prm.declare_entry("cahn hilliard order",
+                        "1",
+                        Patterns::Integer(),
+                        "interpolation order cahn hilliard");
```

Comment:
Cahn hilliard order or do you want to have potential and the other variable order?

Reply:
I'll define two separate orders